ui/app/templates/components/get-credentials-card.hbs

```diff
@@ -6,24 +6,32 @@
     <p class="is-label search-label">{{@searchLabel}}</p>
   </div>
   <p class="sub-text">{{@subText}}</p>
-  <SearchSelect
-    @id={{this.id}}
-    @models={{@models}}
-    @selectLimit="1"
-    @backend={{@backend}}
-    @fallbackComponent="input-search"
-    @shouldUseFallback={{@shouldUseFallback}}
-    @onChange={{action "handleRoleInput"}}
-    @inputValue={{get this.model this.valuePath}}
-    @placeHolder={{@placeHolder}}
-    data-test-search-roles
-  />
-  <input
+  {{#if @renderInputSearch}}
+    <InputSearch
+      @id="search-input-{{@type}}"
+      @onChange={{this.handleInput}}
+      @placeholder={{@placeholder}}
+      data-test-search-roles
+    />
+  {{else}}
+    <SearchSelect
+      @id="search-input-{{@type}}"
+      @models={{@models}}
+      @selectLimit="1"
+      @backend={{@backend}}
+      @fallbackComponent="input-search"
+      @onChange={{this.handleInput}}
+      @placeholder={{@placeholder}}
+      data-test-search-roles
+    />
+  {{/if}}
+  <button
     type="submit"
```

Comment:
A `button` within a form automatically has `type="submit"` so this can be removed!

Reply:
Prettier gets angry if the button doesn't have `type` attribute, i can change it to `type="button"` ?
<img width="856" alt="Screen Shot 2022-09-29 at 10 57 55 AM" src="https://user-images.githubusercontent.com/68122737/193107001-95566e3d-c204-48fb-b842-63368cd39b4a.png">